tests/app/sftp/test_ftp_client.py

```diff
@@ -70,6 +70,19 @@ def test_send_zip_overwrites_if_file_exists_with_different_size(mocks):
     mock_zip_sftp.open.assert_called_once_with('~/notify/' + mocks.mock_remote_filename, mode='w')
 
 
+@freeze_time('2016-01-01T17:00:00')
+def test_send_zip_errors_if_send_zip(mocks):
```

Comment:
I don't understand this test name: a test to send a zip fails because we sent a zip?

Reply:
this test was just a dupe, so ive removed it